lib/ansiblelint/rules/PackageIsNotLatestRule.py

```diff
@@ -29,5 +29,10 @@ class PackageIsNotLatestRule(AnsibleLintRule):
     tags = ['repeatability']
 
     def matchtask(self, file, task):
-        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'package'] and
-                task['action'].get('state') == 'latest')
+        return (task['action']['__ansible_module__'] in ['yum', 'apt', 'dnf', 'homebrew', 'pacman',
+                                                         'homebrew_cask', 'openbsd_package',
```

Comment:
We need to confirm that all of these package managers actually take a version parameter.

I checked homebrew_cask as an example, and it doesn't take version:
http://docs.ansible.com/ansible/homebrew_cask_module.html